src/scripts/main.js

```diff
@@ -22,7 +22,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const createUl = document.createElement('ul');
```

Comment:
Better don't name your variable with a noun in the imperative form. It may be confused with a function
Consider something like 'ulElement' or some other meaningful name